docs/src/wiki-deprecated/ports.md

```diff
@@ -30,21 +30,21 @@ provide powerful wiring constructs described later.
 
 (Chisel 3.2+)
 
-Chisel 3.2+ introduces an API `DataMirror.modulePorts` which can be used to inspect the IOs of any Chisel module, including MultiIOModules, RawModules, and BlackBoxes.
+Chisel 3.2+ introduces an API `DataMirror.modulePorts` which can be used to inspect the IOs of any Chisel module, including Modules, RawModules, and BlackBoxes.
 
```

Comment:
```suggestion
Chisel 3.2 introduced `DataMirror.modulePorts` which can be used to inspect the IOs of any Chisel module (this includes modules in both `import chisel3._` and `import Chisel._`, as well as BlackBoxes from each package).
```

Reply:
but not RawModules?

Reply:
It does include `RawModules`, it's everything 😑 . What do you think the best way to phrase it is? I was trying to avoid just listing every module class